packages/types/src/lib/interfaces.ts

```diff
@@ -770,16 +782,28 @@ export interface GetSessionSigsProps {
   expiration?: any;
 
   //   The chain to use for the session signature.  This is the chain that will be used to sign the session key.  If you're using EVM then this probably doesn't matter at all.
-  chain: any;
+  chain: Chain;
 
-  //   These are the resources that will be signed with the session key.  You may pass a wildcard that allows these session signatures to work with any resource on Lit.  To see a list of resources, check out the docs: https://developer.litprotocol.com/sdk/explanation/walletsigs/sessionsigs/#resources-you-can-request
-  resources: any;
+  /**
+   * An array of resource abilities that you want to request for this session. These will be signed with the session key.
+   *
+   * @example If you want to request the ability to decrypt an access control condition, then you would pass
+   * [{ resource: new LitAccessControlConditionResource('someResource), ability: LitAbility.AccessControlConditionDecryption }]
+   */
+  resourceAbilityRequests: LitResourceAbilityRequest[];
```

Comment:
breaking change, but, as discussed we will proceed like so with heads up to users and that's it.

Reply:
I wanted to introduce `LitResource` and `LitAbility` to better separate those two concepts, rather than have everything be encoded into a single string and parsing in a way that is (IMO) less legible.